index.html

```diff
@@ -60,6 +66,59 @@
         </div>
     </header>
 
+    <div class="slider">
+        <div class="container d-flex flex-column align-items-center pt-9 pt-xs-9 justify-content-xs-around">
+            <div class="col-lg-10 col-sm-12 text-center">
+                <div class="slider__logo pt-7 pb-2 py-xs-7">
+                    <i class="fa-solid fa-list font-xs"></i>
+                </div>
+                <div class="slider__title mt-1 pt-8 pb-4 pt-xs-8 pb-xs-4">
```

Comment:
Không chia nhỏ ra làm padding với margin để làm khoảng cách thế
```suggestion
                <div class="slider__title pt-9 pb-4 pt-xs-8 pb-xs-4">
```